docs/architecture/overview.md

```diff
@@ -1,61 +1,65 @@
 # Miden architecture overview
 
-The Polygon Miden Architecture describes the concepts of how the participants of the network can interact.
+Polygon Miden’s architecture departs considerably from typical blockchain designs to support privacy and parallel transaction execution. 
 
-The architecture reflects the design goals for the rollup:
+In traditional blockchains, state and transactions must be transparent to be verifiable. This is necessary for block production and execution. 
 
-* **High throughput**
-* **Privacy**
-* **Asset safety**
+However, user generated zero-knowledge proofs allow state transitions, e.g. transactions, to be verifiable without being transparent.
 
-## Inspired by the Actor model
-The [Actor Model](https://en.wikipedia.org/wiki/Actor_model) inspired Miden to achieve concurrent and local state changes. In the model, actors are little state machines with inboxes, meaning each actor is responsible for their own state. Actors can send and receive messages to communicate with other actors. Messages can be read asynchronously.
+## Miden design goals
 
-## Core Concepts in Miden
-In Miden, there are accounts and notes which can hold assets. Accounts consume and produce notes in transactions. Transactions describe account state changes of single accounts.
+* [High throughput](https://en.wikipedia.org/wiki/High-throughput_computing): The ability to process a high number of transactions (state changes) over a given time interval.
```

Comment:
I think we should remove that link. High throughput computing in this article doesn't seem to be the same as what we want to express here.